test/tests/performance/__init__.py

```diff
@@ -35,11 +35,73 @@
 # "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
 # trademark licensing policies.
 
+import math
+from math import sqrt
 from ptl.utils.pbs_testsuite import *
 
 
 class TestPerformance(PBSTestSuite):
     """
     Base test suite for Performance tests
     """
-    pass
+
+    def mean(self, lst):
+        """calculates mean"""
+        return sum(lst) / len(lst)
+
+    def stddev(self, lst):
+        """returns the standard deviation of lst"""
+        mn = self.mean(lst)
+        variance = sum([(e - mn) ** 2 for e in lst])
+        return sqrt(variance)
+
+    def check_value(self, res):
+        if isinstance(res, list):
+            for val in res:
+                if isinstance(val, list):
```

Comment:
The result (res) passed here is always a list and never a list of lists. Hence there is no need to check again if it is a list.